docs/dev/style-guide.rst

```diff
@@ -49,6 +49,18 @@ Content
   * ``:guilabel:`<your username>` dropdown``
 * Make sure that **all bullet list items end with a period**, and don't mix periods with no periods.
 
+Linting
+~~~~~~~
+
+ReStructuredText for both the RTD user documentation and this developer documentation is linted with `pre-commit` for broken links,
+whitespace consistency, and other minor issues.
+
+You can fix many linting issues automatically by installing and running `pre-commit` locally. To run it on content you have have already committed, use the ``--from-ref`` and ``--to-ref`` `flags <https://pre-commit.com/#pre-commit-run>`__.
+
+```
+❯ pre-commit run --from-ref <OLDER_COMMIT> --to-ref <RECENT_COMMIT>
+```
+
```

Comment:
I often use `pre-commit run --all-files` if we want to run it against all the existing files, I've never used this from/to syntax.

Reply:
That gives me a bunch of modified files though, where as this just fixes my PR ;-) 

But up to you folks